robottelo/host_helpers/cli_factory.py

```diff
@@ -653,21 +653,24 @@ def setup_org_for_a_custom_repo(self, options=None):
             ) from err
         # Create activation key if needed and associate content view with it
         if options.get('activationkey-id') is None:
-            activationkey_id = self.make_activation_key(
+            activationkey = self.make_activation_key(
                 {
                     'content-view-id': cv_id,
                     'lifecycle-environment-id': env_id,
                     'organization-id': org_id,
                 }
-            )['id']
+            )
         else:
-            activationkey_id = options['activationkey-id']
+            activationkey = self._satellite.cli.ActivationKey.info(
+                {'id': options['activationkey-id']}
+            )
             # Given activation key may have no (or different) CV associated.
             # Associate activation key with CV just to be sure
             try:
                 self._satellite.cli.ActivationKey.update(
                     {
-                        'id': activationkey_id,
+                        'id': activationkey['id'],
+                        'name': activationkey['name'],
```

Comment:
This has been fixed in upstream IIRC.
```suggestion
                        'id': activationkey['id'],
```

Reply:
@vsedmik Oh i see so that was a bug? 
Will push an update removing the name additions